apps/web/src/components/Table/ResponsiveTable/RowSelection.tsx

```diff
@@ -201,45 +203,64 @@ const Actions = ({
                 )}
               </span>
             </Section>
-            <Bullet data-h2-display="base(block)" />
-            <Button onClick={onClear} color="white" mode="inline">
-              {intl.formatMessage({
-                defaultMessage: "Clear<hidden> row selection</hidden>",
-                id: "VHG9Gm",
-                description: "Button text to deselect all table rows",
-              })}
-            </Button>
-            {download?.selection && (
-              <>
-                <Bullet />
-                <DownloadCsv
-                  {...download.selection.csv}
-                  {...actionButtonStyles}
-                >
-                  {download.selection.label ||
-                    intl.formatMessage({
-                      defaultMessage: "Download CSV",
-                      id: "mxOuYK",
-                      description:
-                        "Text label for button to download a csv file of items in a table.",
-                    })}
-                </DownloadCsv>
-              </>
-            )}
-            {print?.onPrint && (
-              <>
-                <Bullet />
-                <Button onClick={print.onPrint} {...actionButtonStyles}>
-                  {print.label ||
-                    intl.formatMessage({
-                      defaultMessage: "Print selection",
-                      id: "KrrW7D",
-                      description:
-                        "Text label for button to print items in a table.",
-                    })}
-                </Button>
-              </>
-            )}
+            <span data-h2-display="base(none) l-tablet(block)">
+              <Bullet data-h2-display="base(none) l-tablet(block)" />
+            </span>
+            <span
+              data-h2-position="base(relative)"
+              data-h2-align-items="base(center)"
+            >
+              <Button
+                data-h2-font-weight="base(400)"
+                data-h2-position="base(relative)"
+                data-h2-display="base(flex)"
+                data-h2-align-items="base(center)"
+                onClick={onClear}
+                color="white"
+                mode="inline"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Clear<hidden> row selection</hidden>",
+                  id: "VHG9Gm",
+                  description: "Button text to deselect all table rows",
+                })}
+              </Button>
+            </span>
+            <span>
+              {download?.selection && (
```

Comment:
I think we move the conditional above the `span` so we don't have an empty `span` tags.